src/main/java/com/sapozhnykov/domain/Cart.java

```diff
@@ -0,0 +1,7 @@
+package com.sapozhnykov.domain;
+
+import java.util.List;
+
+public class Cart {
+    List<Product> products;
```

Comment:
try to avoid work with field directly

Reply:
At the last lesson, Artem said that we would not have a Cart. I removed it from the project.